WORKSPACE

```diff
@@ -1,7 +1,8 @@
 workspace(name = "com_google_dimsum")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
 http_archive(
      name = "com_google_googletest",
      urls = ["https://github.com/google/googletest/archive/master.zip"],
-     strip_prefix = "googletest-master",
```

Comment:
One final question - can you comment on this change here?

Reply:
Two changes had to be made to this file for a successful build.
1. Since Bazel 0.20, use of native http_archive has been deprecated.
2. Since last update to the bazel config, the remote has been renamed to googletest-main.
Ref: [here](https://stackoverflow.com/questions/61163675/error-http-archive-is-not-defined-when-starting-the-local-bazel-server-in-the)